cloudbuild.yaml

```diff
@@ -52,6 +53,26 @@ steps:
         ".",
       ]
 
+  - name: "us.gcr.io/$PROJECT_ID/relay:$COMMIT_SHA"
```

Comment:
We do not use `cloudbuild` anymore, and all the images are built directly in Github actions inside of the docker. See https://github.com/getsentry/relay/blob/d1e4d1284bf07c8ab0846f5c09ab2ef632b498c6/.github/workflows/ci.yml#L228

and later we push the image to GCR as well, here https://github.com/getsentry/relay/blob/d1e4d1284bf07c8ab0846f5c09ab2ef632b498c6/.github/workflows/ci.yml#L277

If we need to upload anything additionally to any of the buckets, we can do from the CI job as well.

Reply:
Production is still using images from `us.gcr.io`; a recent deploy is using `us.gcr.io/sentryio/relay:31b033f3f5f1dca7023bf0cf2b6ddc2f7d604851` but I wasn't able to find what's building them... the only cloudbuild trigger using relay repo in sentryio project is called relay-branch-push which last ran Feb 9.

Reply:
Adjusted.